circleci.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+set -uexo pipefail
+
+HOST_DMD_VER=2.068.2 # same as in dmd/src/posix.mak
+CURL_USER_AGENT="CirleCI $(curl --version | head -n 1)"
+N=2
+
+case $CIRCLE_NODE_INDEX in
+    0) MODEL=64 ;;
+    1) MODEL=32 ;;
+esac
```

Comment:
@wilzbach yes, I had missed that. Glad this is cleared up. Thanks!

@klickverbot Looks like we're good on that, then. But there's that other issue you mentioned - maybe throttling it by using only 1 instance will be more effective for us than running dry halfway through the month.